### DISCUSSION TITLE
How to use GetConnect post with application/x-www-form-urlencoded

### DISCUSSION THREAD

**Discussion by Dufayel** [jonataslaw/getx#1008] [Q&A] [answered]
I'm tring to use GetConnect, but I can't make a post with application/x-www-form-urlencoded
What I did wrong?

`final url = 'http://someurl';
    final body = {'user': xpto, 'pass': xpto};
    final headers = {"Accept": "application/json", "Content-Type": "application/x-www-form-urlencoded"};

    var response= await post(url, body, headers: headers);`

**Comment by eduardoflorence** [comment, marked as answer]
Hi, @Dufayel you should do this:
```dart
final url = 'http://someurl';
final body = 'user=xpto&pass=xpto';
final headers = {"Accept": "application/json"};

final response= await post(
  url, 
  body, 
  contentType: 'application/x-www-form-urlencoded',
  headers: headers
);
```

**Reply by Dufayel** [reply]
Thank You @eduardoflorence, thats works like a charm.

**Reply by oleksii-pimenov** [reply]
Thanks a lot, why did they move contentType  out of regular headers 👎

**Reply by rifkiandriyanto** [reply]
Thank you 🚀 👍

**Comment by bettkipronoh** [comment]
The above works well, but how about if one wants to send special characters like '+'?
